contents/docs/cdp/batch-exports/postgres.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Postgres destination for batch exports
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+Batch exports can be used to export data to a Postgres table.
+
+## Setting up Postgres access
+
+1. Make sure PostHog can access your Postgres database.
+
+> **Note:** Wherever your Postgres database is hosted, make sure the host is set to accept all incoming connections so that PostHog can connect to the database and insert events. If this is not possible in your case, consider exporting data to a different destination (like [S3](./s3.md)) and then setting up your own system for getting data into your Postgres database.
```

Comment:
can we explicitly say here something like `we have no static ips that need to be whitelisted`

Reply:
Yeah, I can add something like that. Give me a commit and you can review again.

Reply:
Gave this a shot in the latest commit 👍